addons/core/stringtable.xml

```diff
@@ -1556,10 +1556,10 @@
                 <Japanese>長距離無線機の標準設定 (NATO):</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_DefaultRadioFrequencies_LR_west_desc">
-                <Original>Define the default LR-settings for the NATO side as a list. e.g.: 99,101,250</Original>
-                <English>Define the default LR-settings for the NATO side as a list. e.g.: 99,101,250</English>
-                <German>Das Festlegen der Standard Langwellenfrequenzen für die NATO-Streitkräfte erfolgt als Liste. z.B.: 99,101,250</German>
-                <Japanese>NATO 側で使われる長距離無線機の標準設定をリストで定義します。(例: 99, 101, 250)</Japanese>
+                <Original>Define the default LR-settings for the NATO side as a list. e.g.: 45, 47, 70</Original>
```

Comment:
why that?

Reply:
I've just lowered the examples for the LR-frequencies.
Everything higher than 87 is not a valid LR frequency.